src/connectors/tidal.js

```diff
@@ -31,9 +31,9 @@ Connector.albumSelector = `${Connector.playerSelector} [class^="infoTable--"] a[
 
 Connector.trackArtSelector = `${Connector.playerSelector} [class^="mediaImageryTrack"] img`;
 
-Connector.currentTimeSelector = `${Connector.playerSelector} [data-test="duration"] [class^="currentTime"]`;
+Connector.currentTimeSelector = `${Connector.playerSelector} [data-test="duration"] [data-test="current-time"]`;
 
-Connector.durationSelector = `${Connector.playerSelector} [data-test="duration"] [class^="duration"]`;
```

Comment:
Can we simplify the selector to `${Connector.playerSelector} [data-test="current-time"]`?